db/migrate/20220114155819_remove_vim_types_from_binary_blob_parts.rb

```diff
@@ -0,0 +1,13 @@
+class RemoveVimTypesFromBinaryBlobParts < ActiveRecord::Migration[6.0]
+  class BinaryBlobPart < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+  end
+
+  def up
+    say_with_time("Removing VimTypes from BinaryBlobParts") do
+      BinaryBlobPart.in_my_region
+                    .where("data LIKE ?", "%!ruby/string:VimString%")
+                    .update_all("data = REGEXP_REPLACE(data, '!ruby/(string|array|hash-with-ivars):Vim(Hash|String|Array)', '!ruby/\\1:\\2', 'g')")
```

Comment:
`BinaryBlobPart#data` is a binary not a string column so this doesn't work:
`PG::UndefinedFunction: ERROR:  function regexp_replace(bytea, unknown, unknown, unknown) does not exist`

Thinking of joning on BinaryBlobs where data_type=yaml then casting to a string